docker/instant-seal-node-with-deployed-schemas.dockerfile

```diff
@@ -0,0 +1,53 @@
+# Docker image for running Frequency parachain node container (with collating)
+# locally in instant seal mode then deploying schemas to that node.
+# Requires to run from repository root and to copy
+# the binary in the build folder.
+
+#This pulls the latest instant-seal-node image
+FROM frequencychain/instant-seal-node as frequency-image
+
+#Creates base image
+FROM --platform=linux/amd64 ubuntu:20.04 AS base
+
+LABEL maintainer="Frequency"
+LABEL description="Frequency collator node in instant seal mode"
+
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+
+#Install git and clone schemas repo
+RUN apt-get update && apt-get install -y git
+
+RUN git clone https://github.com/LibertyDSNP/schemas.git
+
+# Install node-js to base image
+RUN apt-get update && apt-get install -y curl gnupg
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash
+RUN apt-get update && apt-get install -y nodejs
```

Comment:
I was under the impression the line above this install's node from source, isn't this redundant then or am I wrong?